docs/linux-server-setup.md

```diff
@@ -0,0 +1,92 @@
+# Linux Server Setup
+
+| Installation Guide | |
+| :- | :- |
+| This article is a part of the Installation Guide. You can read it alone or click on the previous link to easily move between the steps. |
+| [<< Step 2: Core Installation](core-installation.md) | [Step 4: Database Installation >>](database-installation.md) |
+
+## Extracting DBC, Maps, VMaps & MMaps
+
```

Comment:
where are download links?

Reply:
in server-setup.md the head page before this page

Reply:
Any mac/linux/windows page all have a head page that you need to go through before you can access any specifics